src/models/view/loginUriView.ts

```diff
@@ -62,7 +62,7 @@ export class LoginUriView implements View {
             return null;
         }
         if (this._hostname == null && this.uri != null) {
-            this._hostname = Utils.getHostname(this.uri);
```

Comment:
Can you please verify all locations where this `get hostname()` property is used? The property is explicitly called `hostname`, which is different than a `host`, which we have now changed it to. I think a more proper fix here would be to introduce a new `host` property and then change the clients to reference it on the "view" pages, rather than hostname.

Reply:
That's the route I was going to take - was afraid that I was going to touch something and inadvertently break something somewhere else. 

Will go this route.

Reply:
What I did -

1. Added _host, get host, and get hostOrUri .. mirroring the hostname functionality.
2. When this gets approve - I will modify the following projects:

![image](https://user-images.githubusercontent.com/7233525/87441900-42e2c180-c5b9-11ea-9f14-03ea0735218f.png)

that reference hostnameOrUri on the UI to hostOrUri .. this will make it consistent across the board. 

3. Lastly, will modify mobile project to do the same .. it has a similar function, though not shared in jslib.